bundle/src/main/java/com/adobe/acs/commons/httpcache/store/mem/impl/MemTempSinkImpl.java

```diff
@@ -53,7 +53,7 @@ public InputStream createInputStream() throws HttpCacheDataStreamException {
         if (null != sink) {
             return new ByteArrayInputStream(sink);
         } else {
-            throw new HttpCacheDataStreamException("Nothing available in sink.");
```

Comment:
`HttpCacheDataStreamException` represents an IOException that may happen while reading/writing body content streams . Originally, the problem we wanted to solve was primary to deal with 200. So, supporting other status codes might require this.

Reg. the above said assumption - Any exception thrown in the http cache would let the request pass through for processing.